docs/security.html

```diff
@@ -288,7 +288,7 @@
     };</pre>
                 </li>
                 <li>Pass the JAAS config file location as JVM parameter to each client JVM. For example:
-                    <pre>    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
+                    <pre class="brush: bash;">    -Djava.security.auth.login.config=/etc/kafka/kafka_client_jaas.conf</pre></li>
```

Comment:
I thought it is to be illustrated as part of the cmdline, but it could also be written as part of the script file. So yeah I am fine either way.